docs/cpp/cpp-ide.md

```diff
@@ -4,7 +4,7 @@ Area: cpp
 TOCTitle: Editing
 ContentId: 61D63E54-67E2-4743-B5CB-C6E7F582982A
 PageTitle: Edit and navigate C++ code in Visual Studio Code
-DateApproved: 7/25/2019
+DateApproved: 1/17/2023
```

Comment:
Was the wrong file DateApproved updated?
If cpp-ide.md has been reviewed and doesn't need an update, you can just bump the DateApproved

Reply:
oh, sorry about that...